staging/src/k8s.io/mount-utils/mount_helper_unix.go

```diff
@@ -199,3 +201,49 @@ func PathExists(path string) (bool, error) {
 	}
 	return false, err
 }
+
+var (
+	miBug     bool
+	miBugOnce sync.Once
+)
+
+// kernelHasMountinfoBug checks if the kernel bug that can lead to incomplete
+// mountinfo being read is fixed. It does so by checking the kernel version.
+//
+// The bug was fixed by the kernel commit 9f6c61f96f2d97 (since Linux 5.8).
+// Alas, there is no better way to check if the bug is fixed other than to
+// rely on the kernel version returned by uname.
+func kernelHasMountinfoBug() bool {
```

Comment:
Not related to this coI do not know why mount_helper_unix is very linux specific while we have linux specific build tags.

Reply:
I tried to fix this as well but was not able to do so in this series. Perhaps next time.